src/utils/config.ts

```diff
@@ -40,6 +42,8 @@ export class Config {
         const value = this.Properties[property];
         if (typeof value === 'object') {
             return JSON.stringify(this.Properties[property]);
+        } else if (value == '') {
+            return '*empty*';
```

Comment:
??? first time see this syntax

Reply:
Discord embed fields cannot be empty strings, so if a config value is empty, it will be serialized as _empty_, this method is only used for displaying config values in Discord messages